static/js/debugging.ts

```diff
@@ -86,11 +86,13 @@ export function load_variables(variables: any) {
     for (const i in variables) {
       // Only append if the variable contains any data (and is not undefined)
       if (variables[i][1]) {
-        variableList.append(`<li style=color:${variables[i][2]}>${variables[i][0]}: ${variables[i][1]}</li>`);
+        console.log("HAAI")
```

Comment:
Perhaps this is committed by accident? Could you please remove this line? Otherwise, it looks great.